releases/unreleased/use-native-namespace-packages.yml

```
@@ -0,0 +1,7 @@
+---
+title: Add Poetry support
+category: added
+author: Jose Javier Merchante <jjmerchante@bitergia.com>
+issue: #59
```

```suggestion
issue: 59
```

`#` will be added by the release-tools directly wherever needed.
https://github.com/Bitergia/release-tools/releases/tag/0.3.0

---

I decided to remove the file until we decide to use changelog entries with the new releases.

---

Perfect @jjmerchante :+1: